shells/shells.go

```diff
@@ -15,7 +16,24 @@ const (
 )
 
 func init() {
-	CurrentShell = models.Shell{Name: "fish", GetLastCmd: getLastLineFish}
+	CurrentShell = getShellFromProcess()
+}
+
+func getShellFromProcess() models.Shell {
+	var getLastLine = map[string]func() string{
```

Comment:
can we instead make this a map `string -> shell`, where we initialize all the structs for each shell.

i know this works right now, but if we later want to add something to shell struct, we will need to refactor. plus its cleaner